setup.py

```diff
@@ -45,10 +45,9 @@ def get_readme():
         'Operating System :: OS Independent',
         'Programming Language :: Python',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.5',
-        'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
```

Comment:
scope creep, but wondering if 3.10 support should be a goal too?

Reply:
See the commit message https://github.com/dimagi/django-prbac/pull/69/commits/1eb9aa0a233266c4b7a24682b445ef7c9e0754b2

Reply:
:man_facepalming: